cvmfs/loader.cc

```diff
@@ -736,6 +749,21 @@ int FuseMain(int argc, char *argv[]) {
     fuse_opt_add_arg(mount_options, "-osuid");
     LogCvmfs(kLogCvmfs, kLogStdout, "CernVM-FS: running with suid support");
   }
+
+ if( options_manager->GetValue("CVMFS_CPU_AFFINITY", &parameter) ) {
+     cpu_set_t mask;
+     vector<int> cpus = splitToInts( parameter, ',' );
```

Comment:
I think we should use `SplitString` and `String2UInt64` from the cvmfs string utils instead.